### PR TITLE
update image retrieval schema to include match field

### PR DESCRIPTION
**Pull request by gegeo0**
Update image retrieval schema to include a "match" field

### CODE REVIEW

**Review by harrysummer — APPROVED** [review]
lgtm

**Comment by gegeo0** [comment]
@pj-ms It seems someone else need to review the PR since it's not letting me merge. Can you please take a look?

File: COCO_DATA_FORMAT.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ Here is one example of the json file for the image regression task, where the "t
 
 ## Image retrieval
```
Comment (pj-ms) [inline COCO_DATA_FORMAT.md:113]: This basically make it almost the same with image-text matching task, with a category info. Is it possible to re-use image-text matching?

Comment (pj-ms) [reply COCO_DATA_FORMAT.md:113]: https://github.com/microsoft/vision-datasets/blob/main/COCO_DATA_FORMAT.md#image-text-matching

Comment (gegeo0) [reply COCO_DATA_FORMAT.md:113]: Agree. I'll abandon this PR. In addition, I'll prepare a proposal for a new Image Retrieval dataset type to replace this one and schedule a design review to discuss and align

**Pull request closed without merging**